### ISSUE TITLE
The loss missed

### ISSUE THREAD

**Issue by guoswang** [vsooda/mxnet-wgan#1]
Hi,
 I'm writing a network by mxnet. I find that the loss layer's output is the same as the loss layer's input, I don't l know why? Can you help me. Thanks!

```
density = self.get_VGGNet(datas)
        
Euclidean_loss = mx.sym.LinearRegressionOutput (data = density,label = labels, name = 'Euclidean_loss')
        
if is_train:
      group = mx.sym.Group([mx.sym.BlockGrad(data = density), Euclidean_loss])
else:
      group = mx.sym.Group([density])
```

**Comment by vsooda** [comment]
do you mean there is no loss in `group = mx.sym.Group([density])`

In fact, no loss is needed at the time of prediction.

**Comment by guoswang** [comment]
Sorry that I am not make it clear. I mean that  the loss layer's output is the same as the loss layer's input when I train the net, I don't l know why? Can you help me. Thanks!Please note that the loss function is the Euclidean distance to measure the difference between the density (prediction) and the labels.  The density is a ndarray ,the labels is also a nadarray.
